hw-model/src/model_emulated.rs

```diff
@@ -125,6 +143,10 @@ impl crate::HwModel for ModelEmulated {
         let soc_to_caliptra_bus = root_bus.soc_to_caliptra_bus();
         let cpu = Cpu::new(BusLogger::new(root_bus), clock);
 
+        let mut hasher = DefaultHasher::new();
```

Comment:
nitpick: I think I saw a function above somewhere that does this; maybe call it?

Reply:
nitpick: probably could just be DefaultHasher::new().write(&params.rom).finish()

Reply:
follow up pr.